openeo/rest/datacube.py

```diff
@@ -143,7 +143,7 @@ def load_collection(
         cls,
         collection_id: Union[str, Parameter],
         connection: Optional[Connection] = None,
-        spatial_extent: Union[Dict[str, float], Parameter, None] = None,
+        spatial_extent: Union[Dict[str, float], Parameter, shapely.geometry.base.BaseGeometry, None] = None,
```

Comment:
you also have to update the typehint at https://github.com/Open-EO/openeo-python-client/blob/e106045dfa3c05821c5c3e9d5abd4cd3d7b6514b/openeo/rest/connection.py#L1254-L1257

Reply:
Also, the doc block needs an update for this parameter (which is very short at the moment).

e.g. see the `geometry` argument of `aggregate_spatial`: https://github.com/Open-EO/openeo-python-client/blob/e106045dfa3c05821c5c3e9d5abd4cd3d7b6514b/openeo/rest/datacube.py#L1151-L1162